glean/metrics.yaml

```diff
@@ -120,20 +114,58 @@ page:
       notification_emails:
         - marketing-websites-team@mozilla.com
       expires: never
-  page_event:
```

Comment:
This was not being used, right?

Reply:
I renamed it, now that we have separate events for interaction and non-interaction. We're safe to change it 👍